examples/README.md

```diff
@@ -13,12 +13,11 @@ Please click on each folder for more information.
 
 Each example requires the development environment, set up as follows:
 
-1. Run `docker pull smartcontract/devnet` to download the DevNet image.
-2. Run `../internal/bin/devnet` (or configure direnv `.envrc` to add `chainlink/internal/bin` to your `PATH`)
-3. Run truffle migrations for the Chainlink contracts:
+1. Run `../internal/bin/gethnet` (or configure direnv `.envrc` to add `chainlink/internal/bin` to your `PATH`)
+2. Run truffle migrations for the Chainlink contracts:
```

Comment:
To just run the examples, we don't actually need step 2. These migrations already happen inside each example.